### PR TITLE
Update default smart contract address

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Comment by stbrody** [comment]
gah, this is going to fail tests until https://github.com/ceramicnetwork/js-ceramic/pull/2720 is merged, an RC of it is made, and the CAS codebase gets updated to use the RC

**Review by smrz2001 — APPROVED** [review]
🚀

File: package.json [hunk 1]
```diff
@@ -47,8 +47,8 @@
     }
   },
   "dependencies": {
-    "@ceramicnetwork/common": "^2.20.0-rc.1",
```
Comment (stbrody) [inline package.json]: oh I see why you were asking about this now.  Oops, my bad!  Thanks for fixing!

**Comment by gvelez17** [comment]
hey @stbrody the dependent pr seems to be merged now and i think is already released?

**Comment by gvelez17** [comment]
should we just close this @ukstv ?

**Comment by stbrody** [comment]
Sorry, must have missed this one when I went through lingering old PRs last week.  I'll get this updated today

**Pull request merged**